src/containers/Flow/Flow.test.tsx

```diff
@@ -1,27 +1,39 @@
 import { MockedProvider } from '@apollo/client/testing';
-import { render, waitFor, fireEvent } from '@testing-library/react';
+import { render, waitFor, fireEvent, screen, cleanup } from '@testing-library/react';
 import { MemoryRouter, Route, Routes } from 'react-router-dom';
 import { vi } from 'vitest';
 
 import { getOrganizationLanguagesQuery, getOrganizationQuery } from 'mocks/Organization';
-import { getFlowQuery, filterFlowQuery, updateFlowQuery, copyFlowQuery } from 'mocks/Flow';
+import {
+  getFlowQuery,
+  filterFlowQuery,
+  updateFlowQuery,
+  copyFlowQuery,
+  createFlowQuery,
+  createTagQuery,
+  updateFlowQueryWithError,
+  getFlowQuery2,
+} from 'mocks/Flow';
 import { Flow } from './Flow';
 import { setOrganizationServices } from 'services/AuthService';
 import { getFilterTagQuery } from 'mocks/Tag';
 import { getRoleNameQuery } from 'mocks/Role';
 import userEvent from '@testing-library/user-event';
+import { setErrorMessage, setNotification } from 'common/notification';
 
 setOrganizationServices('{"__typename":"OrganizationServicesResult","rolesAndPermission":true}');
 
 const mocks = [
   ...getOrganizationQuery,
   getFlowQuery,
+  getFlowQuery2,
```

Comment:
I don't think we need getFlowQuery2 since its exactly the same as getFlowQuery. You can use getFlowQuery twice in the mocks